site/examples/check-lists.js

```diff
@@ -116,7 +166,7 @@ const CheckListItemElement = ({ attributes, children, element }) => {
         className={css`
           flex: 1;
           opacity: ${checked ? 0.666 : 1};
-          text-decoration: ${checked ? 'none' : 'line-through'};
+          text-decoration: ${checked ? 'line-through' : 'none'};
```

Comment:
fwiw this fix has already landed in master.

If you have time to rebase your PR, I'll review and land it as this has been open for far too long in my opinion. :)